pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
```

Comment:
Ok, that's something we can improve in future, i will open issue. Is helm also built as static binary? (run ldd on helm)